src/Altinn.AccessManagement.Persistence/DelegationMetadataRepo.cs

```diff
@@ -244,10 +244,10 @@ private async Task<DelegationChange> InsertResourceRegistryDelegation(Delegation
                 { "delegationChangeType", delegationChange.DelegationChangeType },
                 { "altinnAppId", delegationChange.ResourceId },
                 { "offeredByPartyId", delegationChange.OfferedByPartyId },
-                { "coveredByUserId", delegationChange.CoveredByUserId.HasValue ? delegationChange.CoveredByUserId.Value : DBNull.Value },
-                { "coveredByPartyId", delegationChange.CoveredByPartyId.HasValue ? delegationChange.CoveredByPartyId.Value : DBNull.Value },
-                { "performedByUserId", delegationChange.PerformedByUserId.HasValue ? delegationChange.PerformedByUserId.Value : DBNull.Value },
-                { "performedByPartyId", delegationChange.PerformedByPartyId.HasValue ? delegationChange.PerformedByUserId.Value : DBNull.Value },
+                { "coveredByUserId", delegationChange.CoveredByUserId },
+                { "coveredByPartyId", delegationChange.CoveredByPartyId },
+                { "performedByUserId", delegationChange.PerformedByUserId },
```

Comment:
Same ass for app delegation, but for resoure delegation it is also true that only one of performedByUserId and performedByPartyId will have a value

Reply:
Same as AppDelegation. Code is working as before. Dapper accepts int? and does not need a specific DbNull.Value.